src/utils/venue.ts

```diff
@@ -21,7 +21,7 @@ export const canHavePlacement = (venue: Venue): boolean =>
   PLAYA_TEMPLATES.includes(venue.template);
 
 export const peopleByLastSeenIn = (
-  users: Array<WithId<User>> | undefined,
+  users: readonly WithId<User>[] | undefined,
```

Comment:
[nit] I think we could also write this using `?` rather than an explicit `| undefined`

```suggestion
  users?: readonly WithId<User>[],
```